### PR TITLE
MB

### PR DESCRIPTION
**Pull request by alexdiederich**

### CODE REVIEW

**Comment by alexdiederich** [comment]
(sorry if ive ruined everything with the changes and merging, ive got no idea what im doing)

File: docs/aerodromes/Moorabbin (YMMB)/operations.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| Name               | Callsign       | Frequency        | Login Identifier                         |
+| ------------------ | -------------- | ---------------- | ---------------------------------------- |
+| Moorabbin ADC  | Moorabbin Tower  | 118.100         | MB_TWR                        |
+| Moorabbin SMC   | Moorabbin Ground   | 134.250          | MB_GND                       |
+| Moorabbin ATIS        |                | 120.900          | YMMB_ATIS                               |
+
+ 
+In the absence of an SMC controller, ADC shall assume the SMC responsibilities.
+<figure markdown>
+![YSBK Maneuvering Area](img/ysbk_maneuvring_area.png){ width="500" }
```
Comment (tylerthetiletiler) [inline docs/aerodromes/Moorabbin (YMMB)/operations.md:18]: Fix this to YMMB not bankstown

**Review by tylerthetiletiler — COMMENTED** [review]
Delete the DS_Store file

**Review by tylerthetiletiler — COMMENTED** [review]
TODO has a conflict

File: docs/aerodromes/Moorabbin (YMMB)/operations.md (hunk 1 above)
Comment (alexdiederich) [reply docs/aerodromes/Moorabbin (YMMB)/operations.md]: > Fix this to YMMB not bankstown

fixed. no idea where or what ds_store file is. todo conflict resolved (i think)

**Review by chrisg06 — APPROVED** [review]

**Review by zachbf — APPROVED** [review]

**Comment by zachbf** [comment]
generally happy with all except that i'd like to propose a change to a currently common process, that ELW assigns sydney stars to aircraft prior to the transfer, but no descent.
aircraft might prefer more time to prepare for their approaches and by late in ELW the sequence will be stable so delays can start being applied.

**Comment by alexdiederich** [comment]
> generally happy with all except that i'd like to propose a change to a currently common process, that ELW assigns sydney stars to aircraft prior to the transfer, but no descent. aircraft might prefer more time to prepare for their approaches and by late in ELW the sequence will be stable so delays can start being applied.

would it be sufficient to put a note encouraging the BLA controller to handoff as early as possible to GUN? and similarly, telling GUN to ring BLA and request an early handoff with no lateral restrictions if they need to start sequencing straight away? Pretty much how its done IRL

**Comment by mattkelly4** [comment]
I like how BIK (and sub sectors) manage SY and ELW (and sub sectors) manage ML on a milk run. I think that if ELW was issuing STAR for both, we might end up with some situations where the parallels aren't being utilised to their full potential and instead, everyone is just given the main runway and handed to BIK to sequence, who either applies a big delay or reissues STAR to the other runway.

**Review by mattkelly4 — APPROVED** [review]

**Pull request merged**